Source/NSKeyedArchiver.m

```diff
@@ -430,34 +430,49 @@ - (id) _encodeObject: (id)anObject conditional: (BOOL)conditional
 
 @implementation	NSKeyedArchiver
 
++ (NSData *) archivedDataWithRootObject: (id)anObject
+                  requiringSecureCoding: (BOOL)requiresSecureCoding
+                                  error: (NSError **)error
+{
+  NSData *d = nil;
+
+  if (requiresSecureCoding == NO)
```

Comment:
The idea was to call setRequiresSecureCoding: with this value on "a" further down in line 449. And of course any exceptions need to be converted in to NSError.

Reply:
Okay, I will open a "real" secure coding branch for this.   I will do that now.